src/main/drivers/pwm_output.c

```diff
@@ -394,8 +392,17 @@ void pwmWriteDshotCommand(uint8_t index, uint8_t command)
         }
 
         for (; repeats; repeats--) {
-            motor->requestTelemetry = true;
-            pwmWriteDshotInt(index, command);
+            for (uint8_t i = 0; i < motorCount; i++) {
+                if ((i == index) || (index == ALL_MOTORS)) {
+                    motorDmaOutput_t *const motor = getMotorDmaOutput(i);
+                    motor->requestTelemetry = true;
+                    pwmWriteDshotInt(i, command);
+                } else {
+                    // Needed to avoid DMA errors
+                    pwmWriteDshotInt(i, DSHOT_CMD_MOTOR_STOP);
+                }
```

Comment:
This is really just a workaround to prevent problems with `pwmCompleteDshotMotorUpdate` being called without the DMA being set up, which led to faulty DMA output. Ideally DMA would just suspended for these pins, @jflyper, @blckmn  is there an easy way to do this?